tasks/check-for-image-url.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+const https = require('https');
+const http = require('http');
+const path = require('path');
+const url = require('url');
+const fs = require('fs');
+const PluginError = require('plugin-error');
+const through = require('through2');
+const cheerio = require('cheerio');
+
+const { log } = require('./util/util');
+
+const PLUGIN_NAME = 'gulp-inline-images';
+const MIME_TYPE_REGEX = /.+\/([^\s]*)/;
+const INLINE_ATTR = 'inline';
+const NOT_INLINE_ATTR = `!${INLINE_ATTR}`;
+
+function inlineImg(options = {}) {
+  const selector = options.selector || 'img[src]';
+  const attribute = options.attribute || 'src';
+  const getHTTP = options.getHTTP || false;
```

Comment:
@anujpunjani ah cool - so I see this is a new feature then :) 
As I can read, if we would flip this to true there will be a check / inlining for remote urls, which is quite neat.
Sometimes users will want that, sometimes they wont (depending on the size of the image) 

Do you have any suggestion how to inline remote images in a user-friendly way?

I am thinking, we could allow some sort of hint in the url to bypass this option.

So in `conf.js`, the url could have a value of:
- `"url": "https://ex.com/img.png?inline"` to inline as base64
- `"url": "https://ex.com/img.png"` to use the plain url

What do you think?

Reply:
If I understand correctly, in conf.json, we could provide `"inline": true` as a property and use it as a parameter on line 37 of build.js like `.pipe(inlineImg({ getHTTP: confItems[0]["inline"] }))`.

Reply:
Yeah, your idea is even better :) 
Would be awesome to get that in

Reply:
I have added some changes regarding this in the commit [afa48c0](https://github.com/danmindru/responsive-html-email-signature/pull/100/commits/afa48c094ee8d43f2cadb37e508a3470c8e3471a)

Reply:
Awesome! Works great :-) 
Thanks again for the contribution, merged! 🎉

Reply:
This was a new learning experience for me as well. I am glad I could contribute.😄